package.json

```diff
@@ -1,32 +1,71 @@
 {
   "name": "project-health",
   "version": "1.0.0",
-  "description": "[![Build status](https://img.shields.io/travis/PolymerLabs/project-health.svg)](https://travis-ci.org/PolymerLabs/project-health) [![Coverage Status](https://coveralls.io/repos/github/PolymerLabs/project-health/badge.svg?branch=master)](https://coveralls.io/github/PolymerLabs/project-health?branch=master)",
+  "description": "A tool to help Polymer team manage repos.",
   "private": true,
   "scripts": {
-    "clean:client": "cd ./client/ && rm -rf ./node_modules && npm install --only=production",
-    "clean:server": "cd ./server/ && rm -rf ./node_modules && npm install --only=production && cd node_modules && echo num dep files before prune: $(find . -type f | wc -l) && find . -type f -not \\( -name '*.js' -or -name '*.json' -or -name '*.node' \\) -delete && echo num dep files after prune: $(find . -type f | wc -l)",
-    "clean": "npm run clean:client && npm run clean:server",
-    "build:client": "cd ./client/ && npm run build",
-    "build:server": "cd ./server/ && npm run build",
-    "build": "npm run build:client && npm run build:server",
-    "npm-install:client": "cd ./client/ && npm install",
-    "npm-install:server": "cd ./server/ && npm install",
-    "npm-install": "npm run npm-install:client && npm run npm-install:server",
-    "install": "npm run npm-install",
-    "deploy:gcloud": "gcloud app deploy app.yaml --project google.com:project-health-internal --quiet",
-    "deploy": "npm run build && npm run clean && npm run deploy:gcloud && npm run npm-install",
-    "format": "find server/src client/*.ts -name '*.ts' | xargs clang-format --style=file -i",
-    "precommit": "npm run format",
-    "server": "npm run build && node server.js"
+    "generate-gql-types": "apollo-codegen generate \"src/**/*.ts\" --schema \"src/types/github-schema.json\" --target typescript --addTypename --output \"src/types/gql-types.ts\"",
+    "clean": "rm -rf ./build/",
+    "copy": "cd src && cp --parents **/*.{html,json} ../build && cp --parents **/**/*.js ../build && cp -r --parents server/test/replays/**/* ../build",
```

Comment:
`--parents` is not a valid `cp` flag for me. I think it's probably only available in the GNU version, not the BSD version that comes with macOS. Do it some other way?

Reply:
I just switched to a node script after it failed on travis.